README.md

```diff
@@ -34,12 +34,22 @@ mvn clean install
 java -jar target/pazuzu-registry.jar --spring.profiles.active=dev
 ```
 
+How a setup OSX environment
+ * Install [docker toolbox| https://www.docker.com/products/docker-toolbox]
+ * docker-machine start
+ * docker-machine env default
```

Comment:
eval it is short term solution for only current session. It mean if you close your terminal you need to repeat eval every single time